src/System.Net.Http.WinHttpHandler/src/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "Microsoft.Win32.Primitives": "4.0.0",
+    "System.Diagnostics.DiagnosticSource": "4.0.0-rc3-23729",
```

Comment:
Should I update these to 23808? I'm not sure of what the strategy is with checking in specific build dependencies. There is no way to take a dependency on "the version that will be built alongside this project" is there?